lib/rubocop/cop/rspec/let_before_examples.rb

```diff
@@ -43,14 +43,27 @@ class LetBeforeExamples < Base
           }
         PATTERN
 
+        # @!method include_examples?(node)
+        def_node_matcher :include_examples?, <<~PATTERN
+          {
+            #{block_pattern(':include_examples')}
+            #{send_pattern(':include_examples')}
+          }
+        PATTERN
+
         def on_block(node) # rubocop:disable InternalAffairs/NumblockHandler
           return unless example_group_with_body?(node)
+          return if example_group_with_included_examples?(node.body)
```

Comment:
shouldn't we disable just the correction, not the offence?

Reply:
Very well noticed.